<ui version="4.0" >
 <class>logger</class>
 <widget class="QWidget" name="logger" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="spacing" >
    <number>2</number>
   </property>
   <property name="leftMargin" >
    <number>6</number>
   </property>
   <property name="topMargin" >
    <number>4</number>
   </property>
   <property name="rightMargin" >
    <number>6</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTextEdit" name="output_text" >
     <property name="font" >
      <font>
       <family>Monospace</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="acceptDrops" >
      <bool>false</bool>
     </property>
     <property name="tabChangesFocus" >
      <bool>true</bool>
     </property>
     <property name="undoRedoEnabled" >
      <bool>false</bool>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
     <property name="acceptRichText" >
      <bool>false</bool>
     </property>
     <property name="textInteractionFlags" >
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>output_text</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
